extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -353,6 +354,19 @@ export class MigrationCutoverDialog {
 			this._cutoverButton.enabled = false;
 			this._cancelButton.enabled = false;
 			await this._model.fetchStatus();
+			const errors = [];
+			if (this._model.migrationStatus.properties.migrationFailureError?.message) {
+				errors.push(this._model.migrationStatus.properties.migrationFailureError?.message);
+			}
+			if (this._model.migrationStatus.properties.migrationStatusDetails?.fileUploadBlockingErrors) {
+				this._model.migrationStatus.properties.migrationStatusDetails.fileUploadBlockingErrors.forEach(error => errors.push(error));
```

Comment:
Could just do `errors.push(...(this._model.migrationStatus.properties.migrationStatusDetails?.fileUploadBlockingErrors ?? []));` and get rid of the if and forEach